tangelo/linq/gate.py

```diff
@@ -20,6 +20,8 @@
 from typing import Union
 
 from numpy import integer, ndarray, floating
+from sympy import Symbol
```

Comment:
with this, `sympy` is now a main dependency of Tangelo. I just want us to acknowledge that: users can no longer use Tangelo with having sympy in their environment after this change is integrated.

Reply:
Openfermion is already requiring explicitly `sympy` (https://github.com/quantumlib/OpenFermion/blob/master/requirements.txt). However, we could put checkers, but in this case, I don't think it is necessary,

Reply:
Thank you, I had a feeling it was already a dependency :)
I think there's nothing to do here, we're good